crates/typst-library/src/foundations/array.rs

```diff
@@ -815,6 +815,17 @@ impl Array {
     ///
     /// Returns an error if two values could not be compared or if the key
     /// function (if given) yields an error.
+    /// 
+    /// The key function can return a tuple to sort according to the lexicographic order.
```

Comment:
The term "tuple" is not really defined in Typst. It is an array. The "lexicographic" here is also a bit confusing imo since it refers to the tuple but then there are also strings involved. (Is tuple sorting really called "lexicographic"?)

Should also be wrapped at column 80.

Reply:
"Lexicographic" is indeed the appropriate term: https://en.wikipedia.org/wiki/Lexicographic_order

Reply:
I've removed the mention of a tuple and replaced it with array.